src/js/parsers/captions/dfxp.js

```diff
@@ -30,7 +30,7 @@ define([
         for (var i = 0; i < paragraphs.length; i++) {
             var p = paragraphs[i];
             var rawText = (p.innerHTML || p.textContent || p.text || '');
-            var text = strings.trim(rawText).replace(/>\s+</g, '><').replace(/tts?:/g, '');
+            var text = strings.trim(rawText).replace(/>\s+</g, '><').replace(/tts?:/g, '').replace(/<br.*?\/>/g, '\r\n');
```

Comment:
@johanberonius Looks good. We have some unit tests for this already at `dxfp-test.js`, could you add some more to verify the cases you mentioned? Also, feel free to rename the test file to `dfxp-test.js`, just be sure to update it in `test/tests.js` 😸